### ISSUE TITLE
Graphical glitch: oval blue-grey-ish gradient background behind toolbar items.

### ISSUE THREAD

**Issue by johancz** [bwinton/TabCenter#806]
This bug first appeared on the Firefox Developer Edition channel with 52.0a2 (2016-11-21), this is the first Firefox Developer Edition update after the nightly->dev-edition merge.
I can also reproduce the bug on the Nightly channel, but not on the Beta channel.

**Description of bug:** 
An oval blue-grey-ish gradient background appears behind the toolbar items. It bleeds onto the tab scrollbox. See screenshot:
![tab-center-graphics-glitch-bug](https://cloud.githubusercontent.com/assets/1200725/20504199/909612f6-b046-11e6-8652-b2503a2c7534.png)

I ran mozregression which resulted in https://bugzilla.mozilla.org/show_bug.cgi?id=1269046, excerpt from the log:
```
Narrowed inbound regression window from [b48820f4, fa43713a] (3 revisions) to [b48820f4, 995211b2] (2 revisions) (~1 steps left)
Starting merge handling...
Using url: https://hg.mozilla.org/integration/mozilla-inbound/json-pushes?changeset=995211b23a553e1518e8c79570d27f5c4fd0bb86&full=1
Found commit message:
Bug 1269046 part 11: Add reftests for CSS Alignment of absolutely positioned flex children.

MozReview-Commit-ID: FpWhqZdyvBr

2016-11-21T23:43:03: INFO : The bisection is done.
```

**Comment by ericawright** [comment]
I can't seem to reproduce this on either Mac, Ubuntu nor windows 10. Can you let me know what OS you're on and what theme you're using?

**Comment by johancz** [comment]
Right, I probably should've included more info, sorry about that! :)

- I'm running Windows 7 64-bit, all builds of Firefox are 32-bit, including the mozregression builds.
- I'm using the default Firefox theme, and the default Windows 7 theme (Aero).
- On Beta, Dev Edition, Nightly and mozregression I tested with latest "stable" release of TabCenter, v1.26. Bug reproducible on all.
- Only now did it occur to me to test the absolute latest build of TabCenter (i.e. master). So I downloaded the zip, built the .xpi, tested it on Nightly and the bug is still reproducible.

**Comment by ericawright** [comment]
thank you!

**Comment by ericawright** [comment]
This is odd. I'm testing in a VM so it might not be exactly right. But I can't reproduce this at all. 
I'm curious about the coloring of the titlebar as well, it looks almost as if something is behind it and it has some transparency. 
Can you try changing firefox's themes, or disabling tab center? And let me know if the titlebar changes too?

**Comment by johancz** [comment]
_Let me start with some additional technical information:_ I tried disabling hardware acceleration but that did not clear "glitch".

> I'm curious about the coloring of the titlebar as well, it looks almost as if something is behind it and it has some transparency.

The [aero theme](https://en.wikipedia.org/wiki/Windows_Aero) is transparent by default and what you see behind the titlebar are desktop icons:
![tab-center-graphics-glitch-bug-titlebar-blur-desktop-icons](https://cloud.githubusercontent.com/assets/1200725/20604028/9db28684-b264-11e6-9d91-85b895ce2534.png)

The blurry white-ish transparent stuff behind the text in the titlebar is there, I believe, to make the title more legible.

These are the settings I use for the aero-theme:
![windows-7-aero-theme-settings](https://cloud.githubusercontent.com/assets/1200725/20604080/d9bbaf7a-b264-11e6-8113-21df0c18b9b8.png)

> Can you try changing firefox's themes

Sure, this is what I get when I try one of the lightweight themes (found under "Customize" -> "Themes", the theme is called "Pastel Gradient"):
![tab-center-graphics-glitch-bug-with-lightweight-theme](https://cloud.githubusercontent.com/assets/1200725/20604577/662de14c-b267-11e6-9a15-766e55652162.png)
 I feel like this is somehow painting over/overwriting/hiding the graphical glitch with a white background, I base this on the hypothesis that this appears to be a regression from https://bugzilla.mozilla.org/show_bug.cgi?id=1269046. Maybe I should file a bug on bugzilla? This could be more than a styling issue.

> And let me know if the titlebar changes too?

If you look at the screenshot above, it doesn't appear to.

Let me know if you need anything else. :)
